diff --git a/dist/temporary-directory-B83uKxJF.cjs b/dist/temporary-directory-B83uKxJF.cjs
index 2a8f6e8..900c5a1 100644
--- a/dist/temporary-directory-B83uKxJF.cjs
+++ b/dist/temporary-directory-B83uKxJF.cjs
@@ -1 +1 @@
-"use strict";var s=require("node:path"),r=require("node:os");const{geteuid:e}=process,t=e?e():r.userInfo().username,i=s.join(r.tmpdir(),`tsx-${t}`);exports.tmpdir=i;
\ No newline at end of file
+"use strict";var s=require("node:path"),r=require("node:os");const t="workbench",i=s.join(r.tmpdir(),`tsx-${t}`);exports.tmpdir=i;
\ No newline at end of file
diff --git a/dist/temporary-directory-CwHp0_NW.mjs b/dist/temporary-directory-CwHp0_NW.mjs
index 122a35e..c83ee9d 100644
--- a/dist/temporary-directory-CwHp0_NW.mjs
+++ b/dist/temporary-directory-CwHp0_NW.mjs
@@ -1 +1 @@
-import r from"node:path";import o from"node:os";const{geteuid:t}=process,s=t?t():o.userInfo().username,e=r.join(o.tmpdir(),`tsx-${s}`);export{e as t};
\ No newline at end of file
+import r from"node:path";import o from"node:os";const s="workbench",e=r.join(o.tmpdir(),`tsx-${s}`);export{e as t};
\ No newline at end of file
